vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -61,7 +61,7 @@ - (instancetype)initWithStateStoragePath:(NSString *)path
       self.prefs = [[NSMutableDictionary alloc] init];
       self.numberOfAllowableAdsPerDay = kDefaultNumberOfAdsPerDay;
       self.numberOfAllowableAdsPerHour = kDefaultNumberOfAdsPerHour;
-      self.enabled = YES;
+      self.adsEnabled = YES;
```

Comment:
Just realized that we can keep the old naming of the Obj-C method, we just need to change the bridge definition.

In `NativeAdsClientBridge.h` (https://github.com/brave/brave-core/pull/3235/files#diff-99b3ff9eadaeef5ef195cd7d57494aabR22) change `- (bool)isEnabled;` to `- (bool)isAdsEnabled`

In `NativeAdsClient.mm` (https://github.com/brave/brave-core/pull/3235/files#diff-fbfeda143426574b2e4a2bc875b59ddfR19) call this method (`return [bridge_ isAdsEnabled];`)

Then you can revert changes on `isEnabled` in `BATBraveAds.h` and `BATBraveAds.mm`. Only change needed in `BATBraveAds.mm` will be changing the protocol implementation method `isEnabled` → `isAdsEnabled`

Let me know if that works/you have time to do, otherwise I can jump in and fix it

Reply:
Fixed